elastic/integrations
# [cisco_asa] sync with beats cisco module

## What does this PR do?

- imports elastic/beats#26879
- imports elastic/beats#26975
- imports elastic/beats#29698

## Checklist



## How to test this PR locally

``` bash
elastic-package clean
elastic-package build
elastic-package check
elastic-package stack up -d
eval "$(elastic-package stack shellinit)"
elastic-package test
```

## Related issues

- Closes #2116
- Closes #2571
